instrumentation/ratpack/ratpack-1.7/library/src/main/java/io/opentelemetry/instrumentation/ratpack/v1_7/internal/RequestHeaderSetter.java

```diff
@@ -3,13 +3,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.instrumentation.ratpack.v1_7;
+package io.opentelemetry.instrumentation.ratpack.v1_7.internal;
 
 import io.opentelemetry.context.propagation.TextMapSetter;
 import ratpack.api.Nullable;
 import ratpack.http.client.RequestSpec;
 
-enum RequestHeaderSetter implements TextMapSetter<RequestSpec> {
```

Comment:
doesn't need to be public because the only class that uses it was also moved to the same package